Editor/ErrorReporting/ComponentValidation.cs

```diff
@@ -52,6 +52,48 @@ internal static void ValidateAll(GameObject root)
             {
                 component.CheckComponent();
             }
+            
+            CheckVRCFuryCompatibility(root);
+        }
+
+        private static void CheckVRCFuryCompatibility(GameObject root)
+        {
+#if LEGACY_VRCFURY
```

Comment:
For better testing, extract an `internal static` field to indicate whether legacy vrcfury was found. In the tests, you can set this field to simulate the case where vrcfury is present. Note that it must be reset at test teardown to avoid breaking subsequent tests.